Modules/IO/Stimulate/src/itkStimulateImageIO.cxx

```diff
@@ -85,7 +85,7 @@ StimulateImageIO::CanReadFile(const char * filename)
     return false;
   }
 
-  // Check to see if its a spr file
+  // Check to see if it Is a spr file
```

Comment:
Please 🙏  defer such a comment adjustment to a different PR, if you think it's worth it.  This PR is already to big for me to review carefully. And `it Is`n't correct either, anyway. 🤷

Reply:
`PhilipsRECImageIO` is off by default, including in the CI.